### ISSUE TITLE
Toggle is not work after modal view presented.

### ISSUE THREAD

**Issue by joshkim27** [John-Lluch/SWRevealViewController#337]
This UI hierarchy is before present modal view.

![screen shot 2014-09-30 at 5 50 50 pm](https://cloud.githubusercontent.com/assets/7829592/4455419/2284b1ea-487f-11e4-8924-77d2c52639bc.png)

And this UI hierarchy is after present and dismiss modal view.

![screen shot 2014-09-30 at 5 48 26 pm](https://cloud.githubusercontent.com/assets/7829592/4455435/52540042-487f-11e4-8285-2b97b5c5c11c.png)

A position of frontview is now move properly I think.

(iOS8, Xcode 6)

Anybody has same issue?

**Comment by paulius005** [comment]
Yeah, my issue is not exactly the same, but when I segue to a `TabBarViewController` using the modal segue, I am not able to toggle the side menu. The `viewDidLoad` method gets called in the side menu view controller, but the view never shows.
